R/s3path.r

```diff
@@ -4,10 +4,11 @@ s3path <- function() {
   ## The default S3 prefix, for example, `s3://yourbucket/yourprefix/`.
   ## You should set this in everyone's `~/.Rprofile` if
   ## you are using s3mpi to collaborate in a data science team.
-  path <- getOption("s3mpi.path")
+  ## System environment variables are also accepted.
+  path <- getOption("s3mpi.path", Sys.getenv("s3mpi.path"))
```

Comment:
for environment variables the convention would be to do

```r
path <- getOption("s3mpi.path", Sys.getenv("S3MPI_PATH"))
```

that's why I had to write a little helper in this PR https://github.com/kirillseva/mailgunner/pull/3